action.yaml

```diff
@@ -28,6 +28,10 @@ inputs:
   archive:
     description: 'Zip to be used for deployment, instead of archiving a directory.'
     required: false
+  bundle_type:
+    description: >
+      The type of archive: tar, tgz or zip. By default, the extension will
+      be used to auto-detect this value.
```

Comment:
I don't think we should add this parameter. It clouds the list and if we auto-detect it anyway - then doesn't seem needed. Especially since adding this can produce a situation where the archive is a zip, but was told it was a tar.

Less is more for this action so its easily auditable.